_posts/2023-06-08-opensearch-project-newsletter-volume1-issue4.md

```diff
@@ -0,0 +1,79 @@
+* [[Search pipeline] Add script processor](https://github.com/opensearch-project/OpenSearch/pull/7607)
+* [[META] Navigation changes for administrative features](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/4133)
+
+## Good First Issues
+
+* [Grok tags_on_timeout parameter for timing out on pattern matching](https://github.com/opensearch-project/data-prepper/issues/2802)
+* [Replace Lodash library by JS methods](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/4196)
+* [Add data test subjects to table components for easier testing](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/4172)
+* [Separate Display Name From `username_attribute`](https://github.com/opensearch-project/security-dashboards-plugin/issues/1440)
+* [OUI Compliance audit of the `vis_type_tagcloud` plugin](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/4100)
+* [Missing JavaDoc checker to support * for ignoring group of files](https://github.com/opensearch-project/OpenSearch/issues/7264)
+* [Replace dependency management with pipenv](https://github.com/opensearch-project/opensearch-py/issues/392/)
+
+
+
+## Upcoming Events & Meetings
+
+[**Chicago Search Meetup Launch Event with Bonsai, OpenSearch, and Spantree**](https://www.meetup.com/opensearch-project-chicago/events/293331344?sc_channel=sm&sc_campaign=Open_Source&sc_publisher=TWITTER&sc_geo=GLOBAL&sc_outcome=awareness&trk=OpenSearchProject) **June 7, 2023** If you’re near Chicago, join the launch of the Chicago Search Meetup and meet up with your local OpenSearch community! Hosted by [Spantree](https://spantree.net/), this in-person event will include representatives from [Bonsai](http://bonsai.io/) and the [OpenSearch Project](https://opensearch.org/) for networking and discussion, and a presentation from OpenSearch’s David Tippett on the BM-25 search algorithm. [Get registered today](https://www.meetup.com/opensearch-project-chicago/events/293331344?sc_channel=sm&sc_campaign=Open_Source&sc_publisher=TWITTER&sc_geo=GLOBAL&sc_outcome=awareness&trk=OpenSearchProject)!
+
+[**Berlin Buzzwords**](https://2023.berlinbuzzwords.de/) **June 18 - 20** Connect with OpenSearch at Buzzwords, an annual conference exploring search technologies with an emphasis on open-source tools. The OpenSearch Project will have a booth at the conference as well as a presentation showcasing OpenSearch search capabilities. 
+
+[**OpenSearchCon 2023**](https://opensearch.org/OpenSearchCon2023.html) **September 28** The second annual conference for OpenSearch takes place in Seattle with a conference track spanning search, observability, security analytics, and community sessions. Save the date and make your plans to join the OpenSearch community at the Sheraton Grand Seattle, and be sure to check out the [call for presentations](https://opensearch.org/blog/call-for-presentations-bring-your-ideas-to-opensearchcon-2023/)!
```

Comment:
this should link to https://opensearchcon2023.splashthat.com/ with a 'Register Now' call to action